### PR TITLE
Restart execute graph operations in lzy-service

### PR DESCRIPTION
**Pull request by zhvkgj**

### CODE REVIEW

File: lzy/graph-executor/src/main/resources/db/graph/migrations/V1__Init_database.sql [hunk 1]
```diff
@@ -2,19 +2,19 @@ CREATE TYPE graph_execution_status AS ENUM ('WAITING', 'EXECUTING', 'COMPLETED',
 
 CREATE TABLE graph_execution_state
 (
-    workflow_id varchar(255) NOT NULL,
-    workflow_name varchar(255) NOT NULL,
-    user_id varchar(255) NOT NULL,
-    id varchar(255) NOT NULL,
-    error_description text NULL,
-    status graph_execution_status default 'WAITING' NOT NULL,
+    workflow_id                  varchar(255)                             NOT NULL,
```
Comment (imakunin) [inline lzy/graph-executor/src/main/resources/db/graph/migrations/V1__Init_database.sql:5]: let's replace all `varchar` with `text`

File: lzy/graph-executor/src/main/java/ai/lzy/graph/GraphExecutorApi.java [hunk 2]
```diff
@@ -69,11 +78,43 @@ public void execute(GraphExecuteRequest request, StreamObserver<GraphExecuteResp
...
+        }
+
+        var op = Operation.create(
+            request.getUserId(),
+            "Execute graph of execution: executionId='%s'".formatted(request.getWorkflowId()),
+            idempotencyKey,
+            /* meta */ null);
+
+        try {
+            withRetries(LOG, () -> operationDao.create(op, null));
```
Comment (imakunin) [inline lzy/graph-executor/src/main/java/ai/lzy/graph/GraphExecutorApi.java:96]: operation without context
if we fail here we'll have an orphan op

File: lzy/graph-executor/pom.xml [hunk 3]
```diff
@@ -25,6 +25,11 @@
             <artifactId>graph-executor-api</artifactId>
             <version>1.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>ai.lzy</groupId>
+            <artifactId>util-test</artifactId>
+            <version>1.0-SNAPSHOT</version>
```
Comment (imakunin) [inline lzy/graph-executor/pom.xml:31]: `<scope>test</scope>`?

File: lzy/graph-executor/src/main/java/ai/lzy/graph/GraphExecutorApi.java [hunk 4]
```diff
@@ -69,25 +77,75 @@ public void execute(GraphExecuteRequest request, StreamObserver<GraphExecuteResp
...
+        {
+            return;
+        }
+
         final GraphDescription graph = GraphDescription.fromGrpc(request.getTasksList(), request.getChannelsList());
         try {
             graphBuilder.validate(graph);
         } catch (GraphBuilder.GraphValidationException e) {
-            responseObserver.onError(Status.INVALID_ARGUMENT.withDescription(e.getMessage()).asException());
+            var errorStatus = Status.INVALID_ARGUMENT.withDescription(e.getMessage());
```
Comment (imakunin) [inline lzy/graph-executor/src/main/java/ai/lzy/graph/GraphExecutorApi.java:92]: let's log error message here

File: lzy/graph-executor/src/main/java/ai/lzy/graph/api/SchedulerApiImpl.java [hunk 5]
```diff
@@ -26,6 +27,7 @@
 import static ai.lzy.util.grpc.GrpcUtils.newGrpcChannel;
 
 @Singleton
+@Requires(notEnv = "test-mock")
```
Comment (imakunin) [inline lzy/graph-executor/src/main/java/ai/lzy/graph/api/SchedulerApiImpl.java:30]: you may add `@Primary` annotation on the mock scheduler-api implementation, then this `@Requires` may be ommitted

File: lzy/graph-executor/src/main/resources/db/graph/migrations/V1__Init_database.sql [hunk 6]
```diff
@@ -2,19 +2,19 @@ CREATE TYPE graph_execution_status AS ENUM ('WAITING', 'EXECUTING', 'COMPLETED',
...
-    task_executions_json text NOT NULL,
-    current_execution_group_json text NOT NULL,
+    graph_description_json       TEXT                                     NOT NULL,
+    task_executions_json         TEXT                                     NOT NULL,
+    current_execution_group_json TEXT                                     NOT NULL,
 
-    last_updated timestamp NOT NULL,
-    acquired bool NOT NULL,
+    last_updated                 TIMESTAMP                                NOT NULL,
+    acquired                     bool                                     NOT NULL,
```
Comment (imakunin) [inline lzy/graph-executor/src/main/resources/db/graph/migrations/V1__Init_database.sql:17]: BOOL or BOOLEAN :)
let's make it uppercase

File: lzy/graph-executor/pom.xml (hunk 3 above)
Comment (zhvkgj) [reply lzy/graph-executor/pom.xml:31]: it does not work

**Review by imakunin — APPROVED** [review]

**Pull request merged**